EventFilter/Utilities/plugins/TcdsRawToDigi.cc

```diff
@@ -87,20 +88,25 @@ void TcdsRawToDigi::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
     iEvent.getByToken(dataToken_,rawdata);
 
     int nibble=-99;
+    int triggerCount = -99;
     if( rawdata.isValid() ) {
         const FEDRawData& tcdsData = rawdata->FEDData(FEDNumbering::MINTCDSuTCAFEDID);
         if(tcdsData.size()>0){
             evf::evtn::TCDSRecord tcdsRecord(tcdsData.data());
             nibble = (int)tcdsRecord.getHeader().getData().header.nibble;
+            triggerCount = (int)tcdsRecord.getHeader().getData().header.triggerCount;
```

Comment:
triggerCount is apparently uint64_t
https://twiki.cern.ch/twiki/bin/view/CMS/TcdsEventRecord
https://github.com/cms-sw/cmssw/blob/CMSSW_8_2_0/EventFilter/FEDInterface/interface/FED1024.h#L72

it seems inappropriate to save it as an int